developer/testscripts/aatest_ds000114_fmri_localpipeline.m

```diff
@@ -0,0 +1,126 @@
+function aatest_ds000114_fmri_localpipeline(parameterfile, deleteprevious,wheretoprocess)
+
+% This script runs a basic modeling pipeline (preprocessing through secondlevel)
+% using ds000114 motor and line bisection tasks. The data should be dowloaded 
+% prior to running the script
+
+% -------------------------------------------------------------------------
+% init
+% -------------------------------------------------------------------------
+
+aap = aa_test_inittest(mfilename('fullpath'), parameterfile, deleteprevious, wheretoprocess);
+aap = aarecipe(['$AAHOME/developer/testscripts/aatest_ds000114_fmri_localpipeline_prep.xml']);
+
+% -------------------------------------------------------------------------
+% First pre-process the data.
+% -------------------------------------------------------------------------
+
+aap.options.autoidentifystructural_choosefirst = 1;
+aap.options.autoidentifystructural_chooselast = 0;
+
+aap.acq_details.root = '/home/runner/projects';
+aap.directory_conventions.analysisid = 'aatest_ds000114_fmri_local_prep';
+prep_dir = 'aatest_ds000114_fmri_local_prep';
+
+aap.options.remotesymlinks = 1;
+aap.options.NIFTI4D = 1;
+aap.acq_details.numdummies = 4;	
+aap.acq_details.numdummies = 1;
+aap.acq_details.input.correctEVfordummies = 1;
+
+aap.tasksettings.aamod_segment8.writenormimg = 0;
+aap.tasksettings.aamod_segment8.samp = 2;
+aap.tasksettings.aamod_slicetiming.autodetectSO = 1;
+aap.tasksettings.aamod_slicetiming.refslice = 16;
+aap.tasksettings.aamod_smooth.FWHM = 5;
+
+% -------------------------------------------------------------------------
+% BIDS
+% -------------------------------------------------------------------------
+
+aap.acq_details.input.combinemultiple = 1;
+
+% five subjects is about minimum to run a second level model
+
+aap = aas_processBIDS(aap,[],{'finger_foot_lips','line_bisection'},{'sub-01','sub-02','sub-03'});
+
+aa_doprocessing(aap);
+
+aa_close(aap);
+
+% -------------------------------------------------------------------------
+% Part 2 - form a local pipeline connection
+% -------------------------------------------------------------------------
+
+aap = aa_test_inittest(mfilename('fullpath'), parameterfile, deleteprevious, wheretoprocess);
+aap = aarecipe(['$AAHOME/developer/testscripts/aatest_ds000114_fmri_localpipeline_model.xml']);
```

Comment:
Similarly: You MUST use `aa_test_inittest` rather than `aarecipe`.
```matlab
aap = aa_test_inittest([mfilename('fullpath') '_model'], parameterfile, deleteprevious, wheretoprocess);
```